rfcs/detecting_sensitive_textual_content/20230309-implementation_plan_sensitive_terms_list.md

```diff
@@ -0,0 +1,176 @@
+
+### Anticipated questions
+
+1.  Why is the list in a standalone repository, instead of the
+    https://github.com/wordPress/openverse?
+
+    The main reason to keep the list seperate is contributor safety. The list is
+    kept independently from the codebase so that users must explicitly consent
+    to viewing and downloading it. Additionally, there are Search Engine
+    Optimization and blocklist considerations with public GitHub repositories.
+
+    We do not, for example, want students to be unable to access our repository
+    and contribute to it because it is blocked by a school administration's web
+    filter.
+
+2.  Why is the list only in English?
+
+    At the time of writing Openverse doesn't do any translation of the textual
+    contents of indexed media. We also do not translate user-submitted searches.
+    The vast majority of indexed works are in English. Additionally, it would be
+    difficult to validate suggestions for terms in languages not spoken by
+    Openverse maintainers, which to my current knowledge includes English,
+    Spanish, Brazilian Portugese, Turkish, and Russian. English is the only
+    language used by all maintainers.
+
+3.  On what basis will Openverse maintainers use their "discretion" to create
+    this list?
+
+    Until our community has developed its own standards around sensitivity, we
+    can use a few existing documents in the WordPress and user-submitted content
+    ecosystem to guide us:
+
+    - WordPress's page on [etiquette](https://wordpress.org/about/etiquette/) in
+      the project.
+    - WordPress's
+      [code of conduct](https://make.wordpress.org/community/handbook/wordcamp-organizer/planning-details/code-of-conduct/)
+      for in-person events.
+    - YouTube's
+      [community guidelines](https://www.youtube.com/howyoutubeworks/policies/community-guidelines/)
+      offers a decent list of categories for unacceptable types of content. For
+      example, their
+      [thumbnail policy](https://support.google.com/youtube/answer/9229980?hl=en&ref_topic=9282679#zippy=%2Cage-restricted-thumbnails-and-thumbnail-removal)
+      includes some information on types of sexual and violent content that
+      either can't be used or can be used behind a flag comparable to ours.
+      - Additionally, please read
+        [#this-will-not-be-perfect](#this-will-not-be-perfect).
+
+## Implementation Plan
+
+1. Ping a WordPress GitHub account maintainer to request that they create the
+   `WordPress/openverse-sensitive-terms` repository.
+2. Initialize the repository with a `README.md` and a blank
+   `sensitive-terms.txt` file.
+3. Create a PR which initializes the sensitive terms list by copying all terms
+   in the "List of Dirty, Naughty, Obscene, and Otherwise Bad Words" repository.
+   _I am not including the link here to prevent accidental viewing of the
+   repository, but feel free to find it within the GitHub organization
+   "LDNOOBW"._
```

Comment:
First time I see this list so I have to ask #sorryNotSorry :) What was used to select it and why? It's important so we can know how reliable it is. Can we get a bit of information on the State of the Art on these resources?

Reply:
Sure! I can add that.